WordPress/Classes/ViewRelated/NUX/LoginViewController.swift

```diff
@@ -28,6 +28,9 @@ class LoginViewController: NUXViewController, SigninWPComSyncHandler, LoginFacad
         }
     }
 
+    func didChangePreferredContentSize() {
+        styleInstructions()
```

Comment:
For iOS 11, this wouldn't be necessary if `WPStyleGuide.mediumWeightFont()` was returning scaled fonts. I think we can do that and then ignore iOS 10 and lower. Thoughts?

Reply:
I tried, and I could not find a way to make it work as expected without this. The label definitely reacts to changes in the font size, but I couldn't find the way to limit the maximum font size. 😞

Reply:
Ok, right.

It seems like the base styled USED to have a maximum size, too, and now none of them do? Maybe I'm just misremembering.

Anyway, thanks for re-explaining that, I'm ok with this to :shipit: !